src/components/RadioButton/RadioButton.meta.tsx

```diff
@@ -0,0 +1,14 @@
+import { RadioButton } from '.';
+import Registry from '@ui-autotools/registry';
+
+const RadioButtonMetadata = Registry.getComponentMetadata(RadioButton);
+RadioButtonMetadata.nonReactStrictModeCompliant = true;
+
+RadioButtonMetadata.addSim({
+  title: 'render',
+  props: {
+    label: 'label',
+    value: 'value',
+    onChange: () => {},
+  },
+});
```

Comment:
Let's add additional simulations, like `disabled` and `checked`.
You can pretty much copy paste and just add the necessary props

Reply:
for this one i would also need explanation. didn't understand what it does.

Reply:
Sure.
This is a test file that tests a11y and ssr compliancy.
We use a tool called ui-autotools: https://github.com/wix-incubator/ui-autotools

`addSim` is just a test suite actually.
each simulation is a different configuration of the component to test